### PR TITLE
Parsec TPM endorsements

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fix https://github.com/CCC-Attestation/attested-tls-poc/issues/5

### CODE REVIEW

File: scheme/parsec-tpm/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Endorsement Store Interface
+
+## Reference Value
+
+```json
+{
+  "scheme": "PARSEC_TPM",
+  "type": "REFERENCE_VALUE",
+  "attributes": {
+    "parsec-tpm.alg-id": 1,
```
Comment (yogeshbdeshpande) [inline scheme/parsec-tpm/README.md:10]: For consistency with Measurement Description (new change from https://github.com/veraison/services/pull/133 ) we should keep the Algorithm as a string.

**Review by yogeshbdeshpande — COMMENTED** [review]
Let us discuss few of comments tomorrow and close this!

File: scheme/Makefile [hunk 2]
```diff
@@ -6,6 +6,7 @@ SUBDIR += cca-ssd-platform
 SUBDIR += tcg-dice
 SUBDIR += psa-iot
 SUBDIR += tpm-enacttrust
+SUBDIR += parsec-tpm
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply scheme/Makefile:9]: let's do this when the handler is complete (i.e., when we do the evidence part too)

File: scheme/parsec-tpm/corim_extractor.go [hunk 3]
```diff
@@ -0,0 +1,173 @@
+// Copyright 2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package parsec_tpm
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply scheme/parsec-tpm/corim_extractor.go:3]: I am following the convention we established in the scheme folder.

File: scheme/parsec-tpm/README.md (hunk 1 above)
Comment (thomas-fossati) [reply scheme/parsec-tpm/README.md:10]: we need consistency between the evidence and provisioning side of the same scheme.

File: scheme/parsec-tpm/corim_extractor.go [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+
+func (o *ID) FromEnvironment(e comid.Environment) error {
+	if e.Instance != nil {
+		i, err := e.Instance.GetUEID()
+		if err != nil {
+			return fmt.Errorf("could not extract instance-id (UEID) from instance: %w", err)
+		}
+		o.instance = &i
+	}
+
```
[the remark this answers is not in the document]
Comment (thomas-fossati) [reply scheme/parsec-tpm/corim_extractor.go:156]: This could be future work.  Tracked in https://github.com/veraison/services/issues/139

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Pull request merged**